contrib/test_jenkins.sh

```diff
@@ -1120,14 +1116,26 @@ run_test_proto_disable() {
 	run_gtest "default"
 }
 
+run_asan_check() {
+	# build for devel tests and gtest
+	build devel --enable-gtest --enable-asan --without-valgrind
+
+	# all are running gtest
```

Comment:
minor: comments are not really informative here (copy-past)? I'd remove

Reply:
Done.